src/RoleObjects/TreeGridData.js

```diff
@@ -0,0 +1,313 @@
+import _ from 'lodash';
+import { ROLES } from '../Roles';
+import GridData from './GridData';
+
+export default class TreeGridData extends GridData {
+  /**
```

Comment:
Since js doesn't allow for multiple base classes, these setters/getters are bringing over the tree side of https://www.w3.org/TR/wai-aria-1.1/img/rdf_model.svg for available attributes